src/main/perf/SearchPerfTest.java

```diff
@@ -390,6 +444,18 @@ public void run() {
                 mgr.maybeRefresh();
                 reopenCount++;
                 IndexSearcher s = mgr.acquire();
+
+                // An application that uses facets should create new reader states after refreshing.
+                // Here we're demonstrating correct usage, but don't really need the reader states.
```

Comment:
Hmm, don't we need to save and use these reader states whenever SSDV faceting is requested?  Or, is there a hidden cache of the reader states stored somewhere that gets reused when queries run SSDV faceting?

Reply:
Yes, an application would keep track of the reader states computed here and use them later. In this case, I just wanted to show that we should compute them on refresh (even though they are not useful here).